internal/app/app.go

```diff
@@ -21,11 +23,18 @@ import (
 // NewSidecarApp returns a new instance of SidecarApp by applying the specified config params.
 func NewSidecarApp(params *ConfigParams) (*SidecarApp, error) {
 	c := &SidecarApp{params: params}
+	addr := &netlink.Addr{}
 
-	addr, err := netlink.ParseAddr(fmt.Sprintf("%s/32", c.params.IPAddress))
-	if err != nil || addr == nil {
+	ip, err := netip.ParseAddr(c.params.IPAddress)
+	if err != nil {
 		return nil, xerrors.Errorf("unable to parse IP address %q - %v", c.params.IPAddress, err)
 	}
+	
+		addr, err = netlink.ParseAddr(fmt.Sprintf("%s/%d", c.params.IPAddress, ip.BitLen()))
+		if err != nil || addr == nil {
+			return nil, xerrors.Errorf("unable to parse IP address %q - %v", c.params.IPAddress, err)
+		}
```

Comment:
Could you please fix the indentation, e.g. by running `go fmt`?

Reply:
Done :smile: